doc/30_Workspaces.md

```diff
@@ -92,23 +93,23 @@ workspace. See [dashboard](60_Dashboard.md) for details.
 
 ## Shared Workspaces
 Workspaces can be shared from the [dashboard](60_Dashboard.md) or from within the IDE. In the IDE,
-right-click on the user's avatar and choose `Share Workspace`. Share the URL shown in the dialog with
-anyone you want to your workspace with.
+right-click on the user's avatar and choose `Share Workspace`. Provide the URL shown in the dialog to
+anyone you want to share your workspace with.
 
 ![](./images/share-workspace.png)
 
-Beware, anybody with this URL and a Gitpod account will be able to access the workspace as long as it is
-shared and running. By unsharing the workspace, the link becomes useless to anybody but the
+Beware, anybody with this URL and a Gitpod account will be able to access the workspace as long as
+it is shared and running. By unsharing the workspace, the link becomes useless to anybody but the
 workspace owner. All users that are currently looking at your workspace will be shown as avatars in
 the top right corner of the IDE.
 
-All users in one workspace share the same Theia backend and the workspace filesystem.
-Opened editors, terminals, and other UI state are currently not shared.
+All users in one workspace share the Theia backend and the filesystem. Opened editors, terminals,
```

Comment:
This is a matter of taste. I like the `same` here, just to stress that fact. You're right however, it is not strictly neccesary.

Reply:
yep, if it was there alone, I'd leave it there, but ... there's the filesystem too :-)